### PR TITLE
Add MacOS AllControls sample

### PR DESCRIPTION
**Pull request by nosami**

### CODE REVIEW

**Comment by nosami** [comment]
allcontrols macos

File: paket.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 RESTRICTION: == netstandard2.0
 NUGET
   remote: https://www.nuget.org/api/v2
-    FAKE (4.64.13)
+    FAKE (5.0)
```
Comment (dsyme) [inline paket.lock]: I'd recommend just pinning FAKE for now?

**Comment by dsyme** [comment]
This is great. We can merge once CI is green

As an aside, we should do some work to to get these looking "beautiful" by default on both mobile and MacOS.

**Comment by dsyme** [comment]
You're missing a trailing slash on a directory name:

```
build_output/AllControls.MacOS.exe
build_outputAllControls.MacOS.exe
```
in
```

Copying file from "obj/Debug/AllControls.MacOS.exe" to "/Users/travis/build/fsprojects/Elmish.XamarinForms/build_output/AllControls.MacOS.exe".

MMP : error MM0007: The root assembly '/Users/travis/build/fsprojects/Elmish.XamarinForms/build_outputAllControls.MacOS.exe' does not exist [/Users/travis/build/fsprojects/Elmish.XamarinForms/Samples/AllControls/AllControls.MacOS/AllControls.MacOS.fsproj]
```

**Comment by dsyme** [comment]
@nosami I notice the image isn't working, perhaps you didn't give it time to download http://upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Papio_anubis_%28Serengeti%2C_2009%29.jpg/200px-Papio_anubis_%28Serengeti%2C_2009%29.jpg

**Comment by nosami** [comment]
> You're missing a trailing slash on a directory name:

Not sure how that path is being generated, I don't think it is something I have done.

> @nosami I notice the image isn't working in the screenshot, though perhaps you didn't give it time to download http://upload.wikimedia.org/wikipedia/commons/thumb/f/fc/Papio_anubis_%28Serengeti%2C_2009%29.jpg/200px-Papio_anubis_%28Serengeti%2C_2009%29.jpg

I'm not sure what is going wrong there. I just tried it again and left it for a minute. No joy! I'll look into it.

**Pull request closed without merging**